facilitator/src/lib.rs

```diff
@@ -56,6 +56,8 @@ pub enum Error {
     CryptographyUnspecifiedError(String, ring::error::Unspecified),
     PeerValidationError(String),
     ConversionError(String),
+    RusotoError(String),
```

Comment:
Rusoto's APIs (e.g. [get_object](https://rusoto.github.io/rusoto/rusoto_s3/trait.S3.html#tymethod.get_object)) all return errors from distinct enums (e.g. [GetObjectError](https://rusoto.github.io/rusoto/rusoto_s3/enum.GetObjectError.html) vs. [CompleteMultipartUploadError](https://rusoto.github.io/rusoto/rusoto_s3/enum.CompleteMultipartUploadError.html)) which makes it difficult to wrap them without an explosion of enum cases in our own `Error` type. For now I `format!` underlying errors into a `String` and was planning to revisit this once @yuriks's error handling work lands.